wallet/src/actors/app/handlers/create_data_req.rs

```diff
@@ -31,6 +31,7 @@ pub struct CreateDataReqRequest {
     )]
     fee: u64,
     fee_type: Option<FeeType>,
+    required_reward_collateral_ratio: u64,
```

Comment:
Adding this field here would require the client to pass the ratio. Since this should be part of `ConsensusConstants`,  the wallet should already know the value from somewhere else.

```suggestion
```

Reply:
I could not find a reasonable way of getting the consensus constants there, but it's likely I am missing something.

Reply:
Yeah, I found those too and realized I could not really use them. As you suggested, I just added it as a TODO (note that I also had to add the `collateral_minimum` constant).